src/main/main.ts

```diff
@@ -445,6 +447,45 @@ app.on('window-all-closed', () => {
   app.quit();
 });
 
+/**
+ * Checks for updates from the releases page on github, andif there is a new version, sends a message to the main window to display a notification
+ */
+const checkAppUpdate = () => {
+  const options = {
+    hostname: 'api.github.com',
+    protocol: 'https:',
+    path: '/repos/aza547/wow-recorder/releases/latest',
+    method: 'GET',
+    headers: {
+      'User-Agent': 'wow-recorder'
+    }
+  }
+
+  const request = net.request(options);
+  request.on('response', (response) => {
+    let data = '';
+
+    response.on('data', (chunk) => {
```

Comment:
Why do we need this? Are we expecting data to be returned in multiple chunks? I was expecting the response would be short enough that it would be in a single response. 

That being said, if this is just the done thing, it's probably fine - I've never made a HTTP request in JS before (and being robust to all possibilities is obviously a good thing).

Reply:
The response has a [data event](https://www.electronjs.org/docs/latest/api/incoming-message#event-data) that delivers chunks, meaning data might be called multiple times to build the full json object. 
See https://en.wikipedia.org/wiki/Chunked_transfer_encoding